optuna/storages/__init__.py

```diff
@@ -10,6 +10,7 @@
 
 
 __all__ = [
+    "RetryFailedTrialCallback",
```

Comment:
The items seems to be sorted by alphabetical order. Could you place `RetryFailedTrialCallback` after `RedisStorage`?

Reply:
Fixed.

Reply:
Thanks!